Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -1375,9 +1375,15 @@ unsigned MediaPlayerPrivateGStreamerBase::decodedFrameCount() const
 {
     guint64 decodedFrames = 0;
     GstElement* sink = m_fpsSink.get();
+    GRefPtr<GstElement> scopedSink;
 
     if (!sink)
-        sink = m_videoSink ? m_videoSink.get() : getElement(m_pipeline.get(), ElementType::SINK, MediaType::VIDEO);
```

Comment:
Now that sink will be a GRefPtr, keep the old line, removing the `.get()` for `m_videoSink` and keeping `getElement()`